src/ac/api/urls.py

```diff
@@ -4,10 +4,10 @@
 from vng_api_common import routers
 from vng_api_common.schema import SchemaView
 
-# from .viewsets import ExampleViewSet
+from .viewsets import ApplicatieViewSet
 
 router = routers.DefaultRouter()
-# router.register('example', ExampleViewSet)
+router.register('applicatie', ApplicatieViewSet)
```

Comment:
```suggestion
router.register('applicaties', ApplicatieViewSet)
```
we pluralize collections :-)